libs/net/dal/Configuration/ReportSectionConfiguration.cs

```diff
@@ -4,11 +4,15 @@ namespace TNO.DAL.Configuration;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
 using TNO.Entities;
 
-public class ReportSectionConfiguration : BaseTypeConfiguration<ReportSection, int>
+public class ReportSectionConfiguration : AuditColumnsConfiguration<ReportSection>
```

Comment:
Need to override the Description column which requires removing the inheritance and applying the configuration directly on this object.